### PR TITLE
[REVIEW] Pin to cmake 3.18+

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR changes the minimum required CMake version to 3.18 in conda recipe.

### CODE REVIEW

**Review by ajschmidt8 — APPROVED** [review]

**Comment by harrism** [comment]
Please don't push new branches to the master RMM repo. (I assume this was an accident)

**Review by harrism — APPROVED** [review]

**Comment by galipremsagar** [comment]
> Please don't push new branches to the master RMM repo. (I assume this was an accident)

Sorry, Mark. This was by accident. I Will be careful the next time, I will delete the branch once this PR is merged.

**Comment by galipremsagar** [comment]
rerun tests

File: conda/recipes/rmm/meta.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ build:
 
 requirements:
   build:
-    - cmake >=3.14.0
+    - cmake >=3.18.0
```
Comment (kkraus14) [inline conda/recipes/rmm/meta.yaml:23]: I don't think cmake is even required for the Python build, but we should update the `librmm` recipe as well.

Comment (galipremsagar) [reply conda/recipes/rmm/meta.yaml]: Changed it in `librmm` recipe, should we just remove it from `rmm` build recipe?

Comment (kkraus14) [reply conda/recipes/rmm/meta.yaml]: Yea, if we could remove it from the `rmm` recipe that would be nice.

Comment (galipremsagar) [reply conda/recipes/rmm/meta.yaml]: Done

**Review by kkraus14 — APPROVED** [review]

**Comment by kkraus14** [comment]
rerun tests

**Comment by galipremsagar** [comment]
rerun tests

**Comment by galipremsagar** [comment]
rerun tests

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**